apps/docs/stories/architecture-decision-records/0013-why-using-virtual-carousel-item.stories.mdx

```diff
@@ -0,0 +1,44 @@
+import { Meta } from '@storybook/addon-docs'
+
+<Meta title="Docs/Architecture Decision Records/ADR 0013: Why using a virtual carousel item" />
+
+# ADR 0013: Why using a virtual carousel item
+
+🗓️ 2023-08 · ✍️ [@schirrel](https://github.com/schirrel), [@MayaraRMA](https://github.com/MayaraRMA), [@tassioFront](https://github.com/tassioFront) and [@felipefialho](https://twitter.com/felipefialho_)
+
+## Context
+Wrapping up Swiper HTMLElements as Atomium element.
+
+## Problems
+When creating `<atom-carousel>` and its items `<atom-carousel-item>`, the HTML three place the `<atom-carousel-item>` inside `<swiper-element>`, but the swiper needs its children to be `<swiper-slide>`.
+
+Then we need to convert each `atom-carousel-item` into `swiper-slide`.
+
+## Considered Options
+To convert the inner `<atom-carousel-item>` to `<swiper-slide>`, and we have two options:
+
+- Create and `atom-carousel-item` component and inside itself creating the `<swiper-slide>` component on parent and them self deleting from parent.
+- The parent itself convert all children at once.
```

Comment:
Suggestion: better to create an order list to fit with your text: 

```
1. Create and `atom-carousel-item` component and inside itself creating the `<swiper-slide>` component on parent and them self deleting from parent.
2. The parent itself convert all children at once.
```

Reply:
Nice @tassioFront thanks, i've add at this commit [0287647](https://github.com/juntossomosmais/atomium/pull/235/commits/028764756a3ac42e40cf4e9952b7ed1b8daf1e2f)